src/index.ts

```diff
@@ -381,8 +386,23 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
   validation.validateNotBlank(config.apiKey, 'API key required');
   let initializationError: Error | undefined;
   const instance = EppoJSClient.instance;
-  const { apiKey, persistentStore, baseUrl, maxCacheAgeSeconds, updateOnFetch } = config;
+  const { apiKey, persistentStore, baseUrl, maxCacheAgeSeconds, updateOnFetch, forceReinitialize } =
+    config;
   try {
+    if (EppoJSClient.initialized) {
+      if (forceReinitialize) {
+        applicationLogger.warn(
+          'Eppo SDK is already initialized. Since forceReinitialize is true, reinitializing.',
+        );
+        EppoJSClient.initialized = false;
+      } else {
+        applicationLogger.warn(
+          'Eppo SDK is already initialized. Since forceReinitialize is false, returning existing instance.',
```

Comment:
`Eppo SDK is already initialized, skipping reinitialization since forceReinitialize is false`

Reply:
☝️ I like this reword